docs/databalance-README.md

```diff
@@ -0,0 +1,171 @@
+- **Demeaning harms**: When an AI system is actively derogatory or offensive.
+- **Over- and underrepresentation harms**: When an AI system over- or underrepresents some groups of people or may even erase some groups entirely.
+
+**Note**: _Because fairness in AI is fundamentally a sociotechnical challenge, it is often impossible to fully “de-bias” an AI system. Instead, teams tasked with developing and deploying AI systems must work to identify, measure, and mitigate fairness-related harms as much as possible. Data Balance is a tool to help do so, in combination with others._
+
+Data Balance consists of a combination of three groups of measures: Feature Balance Measures, Distribution Balance Measures, and Aggregate Balance Measures.
+
+In summary, Data Balance, when used as a step for building ML models, has the following benefits:
+
+- It reduces costs of ML building through the early identification of data representation gaps that prompt data scientists to seek mitigation steps (such as collecting more data, following a specific sampling mechanism, creating synthetic data, and so on) before proceeding to train their models.
+- It enables easy end-to-end debugging of ML systems in combination with the [RAI Toolbox](https://responsibleaitoolbox.ai/responsible-ai-toolbox-capabilities/) by providing a clear view of model-related issues versus data-related issues.
+
+![Data Balance - All measures](./img/DataBalance-All.png)
+
+## Examples
+
+Within the [Responsible AI Dashboard](https://github.com/microsoft/responsible-ai-widgets/blob/master/notebooks/responsibleaidashboard/tour.ipynb), data balance measures are automatically computed for the dataset if the following criteria is met:
+
+- The task type is `classification`.
+- A non-empty list of categorical features is specified.
+
+Data balance measures are automatically added for the following examples:
+
+- [Model debugging of a census income prediction model (classification)](https://github.com/microsoft/responsible-ai-toolbox/tree/main/notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb)
+
+## Usage
+
+```python
+from raiwidgets import ResponsibleAIDashboard
+from responsibleai import RAIInsights
+
+# categorical features in adult census income dataset (https://archive.ics.uci.edu/ml/datasets/Adult)
+categorical_features = ['workclass', 'education', 'marital-status',
+                        'occupation', 'relationship', 'race', 'gender', 'native-country']
+
+# Since a task type of 'classification' and a non-empty list of categorical features are specified,
+# data balance measures will be automatically added
+rai_insights = RAIInsights(model, train_data, test_data_sample, target_feature,
+                           task_type='classification', categorical_features=categorical_features)
+
+rai_insights.compute()
+
+ResponsibleAIDashboard(rai_insights)
```

Comment:
does this need to mention the flight currently?  I think this feature is under flight, so you need to pass the flight here?

Reply:
You're right! Thanks for catching this